plugins/API/Menu.php

```diff
@@ -26,10 +26,16 @@ public function configureTopMenu(MenuTop $menu)
 
     public function configureUserMenu(MenuUser $menu)
     {
-        $apiUrlParams = $this->urlForAction('listAllAPI', array('segment' => false));
-        $tooltip      = Piwik::translate('API_TopLinkTooltip');
+        $menu->addPlatformItem('General_API',
+            $this->urlForAction('listAllAPI', array('segment' => false)),
+            6,
+            Piwik::translate('API_TopLinkTooltip')
+        );
 
-        $menu->addPlatformItem('General_API', $apiUrlParams, 6, $tooltip);
+        $menu->addPlatformItem('API_Glossary',
```

Comment:
I wonder if a menu item under platform is correct and needed? Under platform is not really correct but there's not really a better place either. Maybe it would be enough to have it linked from `Help` page? Not sure re the importance of this page. Also it would help if someone clicks in the top menu on `Help` as this would be what I would do at least when I was looking for something like this.